travvy/src/services/AuthenticationService.js

```diff
@@ -3,14 +3,23 @@ import axios from 'axios'
 
 export default {
   // It makes post request to add the user information to the database
-    signup(email,name, password){
-        return axios.post('/signup', {params:email,name,password})  
+    async signup(email,name, password){
+        // to begin, set that the user does not exist
+       var userExist = false
+       // create the post request to the database using info from sign up
+        const response = await axios.post('/signup',{params:email,name,password})
+        
         .then(function (response) {
-          console.log(response);
+            // the user exists in the database if the emails match
+            userExist = (email['email'] === response.data['Email'])
         })        
-        .catch(function (error) {
-            console.log(error);
+        .catch(function (error){
+
+            alert(error)
         });
+        console.log(response);
```

Comment:
Good idea to change this and make it actually return true/false instead of undefined. It will make unit testing easier